server/routes/request.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const { 
+  requestList,
+  requestCreate,
+  requestUpdate
+} = require("../controllers/request");
+const { validateRequest } = require("../validate");
+
+router.route("/list").get(protect, requestList);
+router.route("/create").post(protect, requestCreate);
+router.route("/update").put(protect, requestUpdate);
```

Comment:
Regarding your question
> Should I change router to have single path ("/") and different methods to simplify?

I think having the path as "/" for all the routes makes sense.
For `update`, assuming we apply the changes so that it only updates the status, we should change the verb (`PUT` implies we replace the whole resource). Ideally we also indicate which request we want to update by providing a [route parameter](http://expressjs.com/en/guide/routing.html#route-parameters).